### PR TITLE
update typo: Boost_LIBRARYDIR -> BOOST_LIBRARYDIR

### PR DESCRIPTION
**Pull request by kitaniboy**
There is a typo in the CMAKE environment variable; needs to fix "Boost_LIBRARYDIR -> BOOST_LIBRARYDIR".

Without the fix, the custom path for boost library would not work on linux cmake configuration.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -103,7 +103,8 @@ if (WIN32)
     set(Boost_THREADAPI  win32)
 else()
     set(BOOST_INCLUDEDIR "/usr/local/include/boost")
-    set(Boost_LIBRARYDIR "/usr/local/lib/")
+    set(BOOST_LIBRARYDIR "/usr/local/lib")
+
```
Comment (luminousmining) [inline CMakeLists.txt:107]: Remove empty line please.

**Comment by luminousmining** [comment]
Thank you!
Can you remove empty line please and I merge it.

File: CMakeLists.txt (hunk 1 above)
Comment (luminousmining) [reply CMakeLists.txt:107]: https://cmake.org/cmake/help/latest/module/FindBoost.html
BOOST_LIBRARYDIR
Preferred library directory e.g. <prefix>/lib.


Maybe we shoudl remove `/lib` and keep only `set(BOOST_LIBRARYDIR "/usr/local")`

Comment (kitaniboy) [reply CMakeLists.txt]: The <prefix> is set by `BOOST_ROOT`. I think "/usr/local/lib" is the correct default value for `BOOST_LIBRARYDIR`.

**Comment by kitaniboy** [comment]
Okay. The empty line is cleaned up now.

**Pull request merged**